demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -772,6 +772,8 @@ def is_not_valid_display_configuration(self):
 
             elif not is_field_hidden and not configuration_display \
                     and configuration_param['name'] not in ('feedExpirationPolicy', 'feedExpirationInterval'):
+                if configuration_param.get('displaypassword'):
+                    continue
```

Comment:
I think you can expand the `elif` rather than add a new `if` statement

Reply:
done